auditing_standards.md

```diff
@@ -237,3 +237,67 @@ Crates with this criteria contain very dangerous unsafe rust code.
     *   Undefined behavior may be possible under common circumstances.
 *   Most crates that try to be sound but don't quite make the cut go in `ub-risk-3`, not here. These crates are wildly unsafe, and the only time we should use them is when they are a necessary evil.
 *   Everything worse than `ub-risk-3` goes in here and we should do our best to avoid using them.
+
+## Criteria-agnostic guidelines
+
+### Delta audits should describe the final version
+
+In general, delta audits should always take the previous, baseline into
+account.  This means that inspecting just the source code delta should never result
+in lowering the `UB-risk-N` value from the previous audit, or downgrading `crypto-safe` to
+`does-not-contain-crypto`.  OTOH delta audits can lower the UB risk if the delta
+fixes all the issues that were identified in the previous, baseline audit.
```

Comment:
Can you add a note that since deltas only determine which criteria from the source version still apply, you need a non-delta revision to lower the UB risk.

Reply:
Done.

Reply:
@djkoloski, I probably should have shouted to hold off with merging because I made some substantial rewrites.  I think the new text is okay, but the changes to `manual-sources/google3-audits.toml` probably should have required a re-review... :-/  Ooops...